src/modules/commander/PreflightCheck.cpp

```diff
@@ -727,6 +727,9 @@ bool preflightCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &status,
 #elif defined(__PX4_POSIX_RPI)
 	PX4_WARN("Preflight checks for mag, acc, gyro always pass on RPI");
 	checkSensors = false;
+#elif defined(__PX4_POSIX_BBBLUE)
+	PX4_WARN("Preflight checks for mag, acc, gyro always pass on BBBLUE");
+	checkSensors = false;    
```

Comment:
Is this actually necessary?

Reply:
It's convenient for near-term development, but I'd love it if we had at least one non-PX4 platform that did preflight stuff the right way...

Reply:
Try it!

Reply:
To expand on that slightly, most of the value comes from the estimator checks which are already running everywhere. Skipping sensor preflight checks isn't actually as horrible as it initially appears.

Reply:
Hold my be....on second thought, I'll just take your word for it. :-P